src/java.compiler/share/classes/javax/lang/model/util/Elements.java

```diff
@@ -283,15 +283,32 @@ default Set<? extends ModuleElement> getAllModuleElements() {
      * <p> A documentation comment of an element is a comment that
      * begins with "{@code /**}", ends with a separate
      * "<code>*&#47;</code>", and immediately precedes the element,
-     * ignoring white space.  Therefore, a documentation comment
+     * ignoring white space and annotations and end-of-line-comments ({@code "//"} comments).
+     * Therefore, a documentation comment
      * contains at least three "{@code *}" characters.  The text
      * returned for the documentation comment is a processed form of
-     * the comment as it appears in source code.  The leading "{@code /**}"
-     * and trailing "<code>*&#47;</code>" are removed.  For lines
-     * of the comment starting after the initial "{@code /**}",
-     * leading white space characters are discarded as are any
+     * the comment as it appears in source code:
+     * <ul>
+     * <li>The leading "{@code /**}" is removed, as are any
+     * immediately following space characters on that line. If all the
+     * characters of the line are removed, it makes no contribution to
+     * the returned comment.
+     * <li>For subsequent lines
+     * of the doc comment starting after the initial "{@code /**}",
+     * if the lines start with <em>zero</em> or more white space characters followed by
+     * <em>one</em> or more "{@code *}" characters,
+     * those leading white space characters are discarded as are any
```

Comment:
FWIW, I checked `javac` and it allows form-feed in the leading whitespace characters

As an adjective "white space" is normally a single word, at least in JDK.

Reply:
Hmm. java.lang.Character contains both "whitespace" and "white space" in its textual comments.

Reply:
Yes, the difference is typically `whitespace` (adjective) and `white space` noun.  But no matter.